### PR TITLE
float16 to float40

### PR DESCRIPTION
**Pull request by jbaylina**
This Pull Request modifies the circuits to use a 40bit floating point instead of 16bit floating point.

### CODE REVIEW

File: test/rollup-main-L1.test.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const Scalar = require("ffjavascript").Scalar;
 const RollupDB = require("@hermeznetwork/commonjs").RollupDB;
 const Account = require("@hermeznetwork/commonjs").HermezAccount;
 const Constants = require("@hermeznetwork/commonjs").Constants;
-const float16 = require("@hermeznetwork/commonjs").float16;
+const float40 = require("@hermeznetwork/commonjs").float40;
 
 const { depositTx, assertBatch } = require("./helpers/helpers");
```
Comment (krlosMata) [inline test/rollup-main-L1.test.js:13]: Change edge cases to use maximum loadAmountF and amountF allowed:
- https://github.com/jbaylina/circuits-1/blob/master/test/rollup-main-L1.test.js#L150
- https://github.com/jbaylina/circuits-1/blob/master/test/rollup-main-L1.test.js#L190
- https://github.com/jbaylina/circuits-1/blob/master/test/rollup-main-L1.test.js#L201

**Review by krlosMata — CHANGES_REQUESTED** [review]

**Review by krlosMata — APPROVED** [review]
:smile_cat:
